shaas.go

```diff
@@ -88,6 +90,28 @@ func handleExit(res http.ResponseWriter, req *http.Request) {
 	os.Exit(code)
 }
 
+func handleHealth(res http.ResponseWriter, req *http.Request) {
+	responseCode := 200
+	var err error
+	if c := req.URL.Query().Get("responseCode"); c != "" {
+		if responseCode, err = strconv.Atoi(c); err != nil {
+			responseCode = 200
+		}
+	}
+
+	delayMilliSecs := 10
+	if delay := req.URL.Query().Get("delay"); delay != "" {
```

Comment:
same thing about maybe making this work for any endpoint?

Reply:
probably outside of the scope of this work but yeah